### DISCUSSION TITLE
Cyberduck 8.7.2 for Mac not saving changes

### DISCUSSION THREAD

**Discussion by walmar2** [iterate-ch/cyberduck#15564] [Q&A]
I have been using Cyberduck for several years; recently is not saving changes. Mac OS 14.1 Cyber duck 8.7.2

**Comment by tatze96** [comment]
Please elaborate more on the issue:
- Does this issue occur to specific files (file extension, file size) or while editing it within a particular application?
- Is a specific protocol affected?

**Comment by walmar2** [comment]
The issue was corrected when I re-started my computer. So probably not a CyberDuck issue but a computer problem. Thank you for replying.
